File: Project/Java/Moviesion01/src/ca/ryerson/ee/coe848/cmn/Movie.java
package ca.ryerson.ee.coe848.cmn;
public class Movie {

    Integer id;
    String title;
    Director director;
    //other attributes/fields/properties

    public Movie(Integer id, String name, Director director) {
        this.id = id;
        this.title = name;
        this.director = director;
    }
    public Movie() {}

    public Integer getId() {
        return id;
    }
    public void setId(Integer id) {
        this.id = id;
    }
    public String getTitle() {
        return title;
    }
    public void setTitle(String title) {
        this.title = title;
    }
    public Director getDirector() {
        return director;
    }
    public void setDirector(Director director) {
        this.director = director;
    }
    @Override
    public String toString() {
        return "id=" + id + ", name=" + title + ", director=" + director.getName();
    }
}

File: Project/C/Moviesion01/ca.uwindsor.cs.comp3150.cmn/Movie.c
struct Movie {

    int id;
    char title[100];
    struct Director director;
    //other attributes/fields/properties
};

/*
#include "Director.c"
int main() {
  struct Director d;
  d.id = 1;
  d.name = "Stanley Kubrick";
  
  struct Movie m;
  m.id = 1;
  m.title = "2001: A Space Odyssey";
  m.director = d;
  return 0;
}*/
